modules/security-context-constraints-command-reference.adoc

```diff
@@ -84,6 +85,8 @@ Settings:
   Supplemental Groups Strategy: RunAsAny
     Ranges:				<none>
 ----
+<1> Lists which users and service accounts that the SCC is applied to.
```

Comment:
In this case (and the one below), "that" sounds odd to me.
s/Lists which users and service accounts that the SCC is applied to./Lists which users and which service accounts the SCC is applied to.